src/net.rs

```diff
@@ -928,6 +824,35 @@ enum ConnOrigin {
     Dial,
 }
 
+#[allow(missing_docs)]
+#[common_fields({
+    backtrace: Option<snafu::Backtrace>,
```

Comment:
just want to make sure that 

```
    #[snafu(implicit)]
    span_trace: n0_snafu::SpanTrace,
```

was left out on purpose!

Reply:
The error is internal and only debug-printed atm, so the `span_trace` field was unused if I added it. I think it's fine to leave it out for now. Maybe will revisit after #75, for this the error has to change anyway and I might have another look if a span trace is useful to have then.